datafusion/optimizer/src/decorrelate_predicate_subquery.rs

```diff
@@ -442,60 +520,6 @@ mod tests {
         assert_optimized_plan_equal(plan, expected)
     }
 
-    /// Test for IN subquery with additional OR filter
```

Comment:
Why are these tests removed?

Reply:
They were there to verify that the previous code did not handle these kind of subqueries, so their expected output needed to be updated. But the new sqllogictests provide the same coverage, so opted for removing these instead of updating them.